File: Actividad 2/actividad212.py
nom = input("Ingrese nombre: ")
nht = int(input("Ingrese horas trabajadas: "))
vhn = float(input("Ingrese el valor de hora de trabajo: "))
het = 0
hees = 0
if nht > 40:
    
    hees = nht - 40
    if hees > 8:
        het = hees - 8
        het = het * vhn * 3
        hees = 8 * vhn * 2
    else:
        hees = hees * vaho * 2
    
    salario = (40 * vhn) + hees + het

else: 
    salario = vhn * nht


print(f"El trabajador {nom} devengo: {salario}")

File: Actividad 2/actividad212.java
import java.util.Scanner;

public class actividad212 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Ingrese nombre: ");
        String NOM = scanner.nextLine();

        System.out.print("Ingrese horas trabajadas: ");
        int NHT = Integer.parseInt(scanner.nextLine());

        System.out.print("Ingrese el valor de hora de trabajo: ");
        double VHN = Double.parseDouble(scanner.nextLine());

        int HET = 0;
        int HEES = 0;
        double salario;

        if (NHT > 40) {
            HEES = NHT - 40;
            if (HEES > 8) {
                HET = HEES - 8;
                HET = (int) (HET * VHN * 3);
                HEES = (int) (8 * VHN * 2);
            } else {
                HEES = (int) (HEES * VHN * 2);
            }

            salario = (40 * VHN) + HEES + HET;
        } else {
            salario = VHN * NHT;
        }

        System.out.println("EL TRABAJADOR " + NOM + " DEVENGÓ: $" + salario);
    }
}
